### ISSUE TITLE
MSON block descriptions and markdown in descriptions could be improved

### ISSUE THREAD

**Issue by jamesramsay** [apiaryio/api-blueprint#239]
It seems MSON block descriptions don't render at all at the moment?

https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md#41-block-description

It would be really great to support longer descriptions to help the humans who don't read the JSON schema. Also, markdown bold and italics definitely help highlight certain critical pieces of information.

```markdown
FORMAT: 1A

# Blueprint

## Item [/item]

+ Attributes
    - id: 1 (number) - This description renders nicely
    - name: example (string) - This one doesn't _render italics_ or **bold**
    - short_description: foobar (string) - First line renders
        This doesn't render at all!
    - description: foobar (string)
        This doesn't render at all!

### Get Item [GET]

+ Response 200 (application/json)

    + Attributes (Item)
```

**Comment by zdne** [comment]
That is correct. It should be addressed in the – now infamous – #191

**Comment by pksunkara** [comment]
I think this can be closed as a duplicate of #191
